crates/bevy_window/src/window.rs

```diff
@@ -95,8 +95,27 @@ impl NormalizedWindowRef {
 /// When the [`Window`] component is added to an entity, a new window will be opened.
 /// When it is removed or the entity is despawned, the window will close.
 ///
+/// Primary window (the window and the component) is spawned by default
+/// by [`WindowPlugin`](crate::WindowPlugin) and is marked with [`PrimaryWindow`] component.
+///
 /// This component is synchronized with `winit` through `bevy_winit`:
 /// it will reflect the current state of the window and can be modified to change this state.
+///
+/// # Example
+///
+/// System to change cursor:
```

Comment:
```suggestion
/// Because this component is synchronized with `winit`, it can be used to perform OS-integrated windowing operations.
/// For example, here's a simple system to change the cursor type:
```